apps/learner/src/lib/components/Button/LinkButton.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+  import type { HTMLAnchorAttributes } from 'svelte/elements';
+
+  export interface Props extends Omit<HTMLAnchorAttributes, 'aria-disabled'> {
+    /**
+     * The variant of the button.
+     *
+     * @default 'primary'
+     */
+    variant?: 'primary' | 'secondary';
```

Comment:
Are we setting the `(Link)Button` variant based on hierarchical importance or color/theme? i.e. is it intended to represent the main action on this screen?

Meaning when users see this, is our intention to let users focus on clicking `'Play'` first?
<img width="733" height="289" alt="Screenshot 2025-07-29 at 4 25 00 PM" src="https://github.com/user-attachments/assets/8f9b9ccc-bd23-4089-a0a4-911e1b7011a7" />


If it is, can keep the variant to `primary/secondary.` Else if it's not meant for hierarchal importance, i would choose `dark/light` variant instead.